krcal/core/io_functions.py

```diff
@@ -50,3 +52,96 @@ def compute_and_save_hist_as_pd(values     : np.array           ,
     out_file.put(hist_name, table, format='table', data_columns=True)
 
     return
+
+
+
+def compute_and_save_hist_as_pdf(values     : np.array           ,
+                                 out_file   : str                ,
```

Comment:
Here and throughout the PR, np.array type is np.narray. I know this is wrong so far in the repo, but maybe it is an excellent time to start fixing it!